docs/components/styling.md

```diff
@@ -233,6 +233,66 @@ By specifying "inner-text" as the value of the `exportparts` attribute, elements
 <outer-component />
 ```
 
+## Style Modes
+
+Component Style Modes enable you to create versatile designs for your components by utilizing different styling configurations. This is achieved by assigning the styleUrls property of a component to a collection of style mode names, each linked to their respective CSS files.
+
+### Example: Styling a Button Component
+
+Consider a basic button component that supports both iOS and Material Design aesthetics:
+
+```tsx
+@Component({
+  tag: 'simple-button',
+  styleUrls: {
+    md: './simple-button.md.css', // styles for Material Design
+    ios: './simple-button.ios.css' // styles for iOS
+  },
+})
+export class SimpleButton {
+  // ...
+}
+```
+
```

Comment:
Can we add a sentence or two about the content of `styleUrls` above? I think there might be room to explain it a bit more/describe what's happening here. What do you think about:
```suggestion

In the example above, two different modes are declared.
One mode is named `md` (for 'Material Design') and refers back to a Material Design-specific stylesheet.
Likewise, the other is named `ios` (for iOS) and references a different stylesheet for iOS-like styling.
Both stylesheets are relative paths to the file that declares the component.
While we have chosen short names in the above example, there's no limitation to the keys used in the `styleUrls` object.

```

Reply:
LGTM 👍